### PR TITLE
automated: linux: add video output comparison test

### PR DESCRIPTION
**Pull request by mwasilew**
The test takes a screenshot from video output and compares to the reference image. It is intended to test HDMI output from DUT.

### CODE REVIEW

**Comment by mwasilew** [comment]
@roxell it looks like the linter check fail is not related to this patch. Could you take a look?

File: automated/linux/video/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM python:3
+
+COPY requirements.txt /home/
+RUN python3 -m pip install opencv-python scikit-image requests
```
Comment (bhcopeland) [inline automated/linux/video/Dockerfile:4]: why don't you have all this in requirements.txt and install from there?

Comment (mwasilew) [reply automated/linux/video/Dockerfile:4]: It takes 1.5 hours. Both Dockerfile and requirements.txt are not needed for the test. They're added if someone would like to a) build the container to run the test in LAVA, b) run the test locally.

Comment (mwasilew) [reply automated/linux/video/Dockerfile:4]: ah, and using requirments.txt in Dockerfile will be tricky. If the requirements.txt changes, the layer cache will not be invalidated.

**Review by roxell — APPROVED** [review]

**Pull request merged**